.readthedocs.yaml

```diff
@@ -15,4 +15,7 @@ sphinx:
 
 python:
    install:
-    - requirements: requirements-doc-3.11.txt
+    - method: pip
+      path: .
+      extra_requirements:
+        - doc
```

Comment:
Do we want to point to the pinned version here instead. This is based on [RTD recommendations](https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html).

Reply:
Much as I struggle to be sold on reproducible builds, I can't argue against following official guidelines.